src/index.html

```diff
@@ -4,14 +4,201 @@
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0,
+      minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      onsubmit="onSubmit()"
+      method="POST"
+    >
+      <fieldset class="distanceFull">
+        <legend>
+        Personal information
+        </legend>
+        <div class="distanceText">
+          <label for="Surname">
+            Surname:
+            <input
+            type="text"
+            name="Surname:"
+            autocomplete="off"
+            id="Surname"
+          >
```

Comment:
Still, indentations are not quite correct.
Also, id and name should not start with Capital letter.
Fix it all over your document.
```suggestion
            <input
              type="text"
              name="surname:"
              autocomplete="off"
              id="surname"
            >
```